### ISSUE TITLE
Driver manager cant download driver when use xdist plugin for pytest

### ISSUE THREAD

**Issue by KursLabIgor** [SergeyPirogov/webdriver_manager#54]
In case if we run tests in two or more threads produce error ```pythonOSError: [Errno 17] File exists: '/root/.wdm/chromedriver/2.41/linux64'```
Reproduce only in docker containers based on ubuntu 16.04. 
Reproduce time to time. 
```python
d = webdriver.Chrome(executable_path=ChromeDriverManager().install(),
/usr/local/lib/python2.7/dist-packages/webdriver_manager/chrome.py:19: in install
    bin_file = self._file_manager.download_driver(self.driver, path)
/usr/local/lib/python2.7/dist-packages/webdriver_manager/cache.py:73: in download_driver
    zip_file = self._download_file(driver, path)
/usr/local/lib/python2.7/dist-packages/webdriver_manager/cache.py:105: in _download_file
    self.create_cache_dir(driver_path)
/usr/local/lib/python2.7/dist-packages/webdriver_manager/cache.py:26: in create_cache_dir
    os.makedirs(driver_path)```
name = '/root/.wdm/chromedriver/2.41/linux64', mode = 511

    def makedirs(name, mode=0777):
        """makedirs(path [, mode=0777])
    
        Super-mkdir; create a leaf directory and all intermediate ones.
        Works like mkdir, except that any intermediate path segment (not
        just the rightmost) will be created if it does not exist.  This is
        recursive.
    
        """
        head, tail = path.split(name)
        if not tail:
            head, tail = path.split(head)
        if head and tail and not path.exists(head):
            try:
                makedirs(head, mode)
            except OSError, e:
                # be happy if someone already created the path
                if e.errno != errno.EEXIST:
                    raise
            if tail == curdir:           # xxx/newdir/. exists if xxx/newdir exists
                return
>       mkdir(name, mode)
E       OSError: [Errno 17] File exists: '/root/.wdm/chromedriver/2.41/linux64'
```

**Comment by MarkBrook** [comment]
Will be fixed for python 3.2+. Considering that webdriver_manager now supports only python 3.5+, this will work.

**Comment by SergeyPirogov** [comment]
@KursLabIgor fixed
